### PR TITLE
[EE-27556] expose the `onFocus` and `onBlur` of the wix-ui-core/RadioButton component

### PR DESCRIPTION
**Pull request by urig-wix**
expose the `onFocus` and `onBlur` of the wix-ui-core/RadioButton component

### CODE REVIEW

File: packages/wix-ui-core/src/components/radio-button/RadioButton.tsx [hunk 1]
```diff
@@ -163,13 +172,15 @@ export class RadioButton extends React.Component<
...
+  onFocus = (event: React.FocusEvent<HTMLInputElement>) => {
     this.setState({ focused: true, focusVisible: !this.focusedByMouse });
+    this.props.onFocus && this.props.onFocus({ value: this.props.value, ...event });
   };
 
-  onInputBlur = () => {
+  onInputBlur = (event: React.FocusEvent<HTMLInputElement>) => {
     this.setState({ focused: false, focusVisible: false });
     this.focusedByMouse = false;
+    this.props.onBlur && this.props.onBlur({ value: this.props.value, ...event });
```
Comment (jonathanadler) [inline packages/wix-ui-core/src/components/radio-button/RadioButton.tsx:183]: Nice work 💪 

However, I'm not really sure the `value` is needed in this case.
It also doesn't align with the standard native focus/blur event.

wdyt?

Comment (urig-wix) [reply packages/wix-ui-core/src/components/radio-button/RadioButton.tsx]: Removed `value` :)

File: packages/wix-ui-core/src/components/radio-button/RadioButton.tsx [hunk 2]
```diff
@@ -163,13 +167,15 @@ export class RadioButton extends React.Component<
     this.props.onHover({ value: this.props.value, ...event });
   };
 
-  onFocus = () => {
+  onFocus = (event: React.FocusEvent<HTMLInputElement>) => {
     this.setState({ focused: true, focusVisible: !this.focusedByMouse });
+    this.props.onFocus && this.props.onFocus({ ...event });
```
Comment (jonathanadler) [inline packages/wix-ui-core/src/components/radio-button/RadioButton.tsx:172]: Could be changed to
```suggestion
    this.props.onFocus && this.props.onFocus(event);
```
but not critical 😊

File: packages/wix-ui-core/src/components/radio-button/RadioButton.tsx [hunk 3]
```diff
@@ -163,13 +167,15 @@ export class RadioButton extends React.Component<
...
   };
 
-  onInputBlur = () => {
+  onInputBlur = (event: React.FocusEvent<HTMLInputElement>) => {
     this.setState({ focused: false, focusVisible: false });
     this.focusedByMouse = false;
+    this.props.onBlur && this.props.onBlur({ ...event });
```
Comment (jonathanadler) [inline packages/wix-ui-core/src/components/radio-button/RadioButton.tsx:178]: As well
```suggestion
    this.props.onBlur && this.props.onBlur(event);
```

**Review by jonathanadler — APPROVED** [review]
Nice 👌

**Review by jonathanadler — APPROVED** [review]

**Pull request merged**